### DISCUSSION TITLE
Disabling fiat onboarding features for restricted geo-locations

### DISCUSSION THREAD

**Discussion by LPX55** [shapeshift/web#1578] [Polls]
1. Certain countries, like South Korea, has blocked all fiat onboarding that is not on their list of approved exchanges (4 exchanges based in Korea as of this writing), and services like Banxa, OnJuno, and Gem, despite what they say, are not a viable onboarding option. Even though Gem lists South Korea as an approved/supported country, I have personally checked and, unsurprisingly, all banks denied the transaction. 

2. Users from countries that are on the list of sanctions, or just not supported in general by the onboarder would also see this button but ultimately fail to acquire crypto through these services.

Now, a few concerns (CONS):

1. Blocking features via geo-location is always a bad idea as users can be on a VPN, or is simply traveling.
2. Blocking features via locale settings is also a bad idea, as it doesn't necessarily detect the users location.
3. Someone in an unsupported country could be holding bank accounts & cards from supported countries and ultimately go through with a successful transaction.

On the flip side, some reasons for disabling or removing the button completely for unsupported countries (PROS):

1. It's super frustrating for users to be given false hope; in my case, everything with Gem onboarding went smoothly, until the very last moment (after I have submitted all the information and CC info). Transaction was reversed instantly by the banks I have tried.
2. This may be confusing, especially for crypto newcomers, and may leave the platform with a not-so-good experience.

I am torn on the direction that this should take, so I was hoping to get input from the community.

**Comment by gomesalexandre** [comment]
Regarding all your cons, I can speak from the mindset of someone traveling regularly:

- I don't expect things to work the same way as they do at "home" when living in a different country
- I don't expect things to work the same way as they do at "home"/<the country I live in> when I travel
- Being detected as being in X country even though you have Y bank card is unfortunate, but that's pretty much the experience I'd expect, usually because of legislation

Unfortunately there isn't a way to vote multiple options but as a user, I'd be happy with either:
- The fiat onboarding being disabled altogether
- A message telling me that it's not supported in the country I'm in (if I'm e.g holding a card from a different country, then I can just continue normally)

I'd be actually more inclined towards the second option. The experience I expect as a user is probably suboptimal because that's just how it is in most services, so why not go and make it nicer here!
